stl/inc/experimental/deque

```diff
@@ -25,13 +24,13 @@ namespace experimental {
 
         // FUNCTION TEMPLATE erase_if
         template <class _Ty, class _Alloc, class _Pr>
-        void erase_if(deque<_Ty, _Alloc>& _Cont, _Pr _Pred) { // erase each element satisfying _Pred
+        _DEPRECATE_EXPERIMENTAL_ERASE void erase_if(deque<_Ty, _Alloc>& _Cont, _Pr _Pred) {
             _Erase_remove_if(_Cont, _Pass_fn(_Pred));

```

Comment:
Should we qualify these calls now that they're crossing namespaces?

Reply:
I haven't `_STD` qualified ugly names I call from `std::ranges`. I don't recall `std::filesystem` doing so either. (Looks at `<filesystem>`, sees ~160 occurrences of `_STD`) I could be mistaken.

Reply:
I see no occurrences of `_STD _` in `<filesystem>`.

I believe that our current convention is to avoid qualifying `_Ugly` function calls except when necessary (e.g. `stdext` isn't within `std`). We may change that convention in response to #140, but for now, the code is consistent as-is.

Reply:
Sounds good to me :)